tests/models/test_dagwarning.py

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+import unittest
+from unittest.mock import MagicMock
+
+from sqlalchemy.exc import OperationalError
+
+from airflow.models.dagwarning import DagWarning
+
+
+class TestDagWarning(unittest.TestCase):
```

Comment:
We now use pytest for our tests. With pytest, you have access to `session` fixture

Reply:
Are there any good examples of using the shared session fixture?

Reply:
Check this: https://github.com/apache/airflow/blob/378c8faba19d21b29abfff1705a8e7624aeabdc4/tests/models/test_taskinstance.py#L507

Reply:
@ephraimbuddy What do you think of the tests?